src/logger/log_entry.py

```diff
@@ -120,14 +130,14 @@ class SetSignalLogEntry(LogEntry):
     class Schema(LogEntry.Schema):
         """The marshmallow schema for the SetSignalLogEntry model."""
 
-        signal_id = marsh.fields.String(required=True)
+        signal_id = marsh.fields.UUID(required=True)
         state_before = marsh.fields.Integer(required=True)
         state_after = marsh.fields.Integer(required=True)
 
         def _make(self, data: dict) -> "SetSignalLogEntry":
             return SetSignalLogEntry(**data)
 
-    signal_id = CharField(null=False)
+    signal_id = TextField(null=False)
```

Comment:
You can use `UUIDField` here 😇

Reply:
Did not know, it existed, good to know :) I'll change it

Reply:
Perfect 🥳